htdocs/lang/de/ocstyle/newcache.tpl.php

```diff
@@ -99,8 +99,15 @@ function toggleAttr(id)
     {
         if (maAttributes[i][0] == id)
         {
-            if (maAttributes[i][1] == 0)
+            if (maAttributes[i][1] == 0) {
                 maAttributes[i][1] = 1;
+                for (a = 0; a < maAttributes[i][4].length; a++)
```

Comment:
missing {

Reply:
It's not missing, because so far we don't have code style rules for Javascript. Many JS control structures in the OC code are unbracketed.

Do you want to introduce JS coding rules?

Reply:
Damn js in tpl.php files 

![why](https://media.giphy.com/media/l2JhtKtDWYNKdRpoA/giphy.gif)

> Do you want to introduce JS coding rules?

no - ignore my comments